DMCompiler/DM/Visitors/DMProcBuilder.cs

```diff
@@ -318,6 +319,22 @@ class DMProcBuilder {
             _proc.DestroyEnumerator();
         }
 
+        //Generic infinite loop, while loops with static expression as their conditional with positive truthfullness get turned into this as well as empty for() calls
```

Comment:
I don't see where you did this for `while` loops.

Reply:
aa i forgot to do this for while loops gimme a moment